ReactWindows/ReactNative.Shared/PlatformServices/ServiceResolver.cs

```diff
@@ -0,0 +1,13 @@
+namespace ReactNative.PlatformServices
+{
+    /// <summary>
+    /// Singleton for platform-specific service provider.
+    /// </summary>
+    public static class ServiceResolver
+    {
+        /// <summary>
+        /// The current platform service provider.
+        /// </summary>
+        public static IServiceProvider Instance { get; } = new CurrentPlatformServiceProvider();
```

Comment:
Thinking we should hide the instance and expose a static GetService<T>() here

Reply:
Even better